src/distributions/mod.rs

```diff
@@ -80,6 +82,15 @@ pub trait IndependentSample<Support>: Sample<Support> {
     fn ind_sample<R: Rng>(&self, &mut R) -> Support;
 }
 
+/// DEPRECATED: Use `distributions::uniform` instead.
+#[deprecated(since="0.5.0", note="use uniform instead")]
+pub mod range {
+    pub use distributions::uniform::Uniform as Range;
+    pub use distributions::uniform::UniformInt as RangeInt;
+    pub use distributions::uniform::UniformFloat as RangeFloat;
+    pub use distributions::uniform::SampleUniform as SampleRange;
```

Comment:
I don't think there's any point exporting the last three here because `RangeInt` and `RangeFloat` were never in a release and `SampleRange` is totally different.

Reply:
I agree about `Range*` and removed them. (I was not aware they were not released.)

However, what do you mean about `SampleRange`? It was public (although not supposed to be used), so removing is a breaking change, hence the deprecated reexport.

Reply:
Compare [the old](https://docs.rs/rand/0.4.2/rand/distributions/range/trait.SampleRange.html) and [the new](https://docs.rs/rand/0.5.0-pre.0/rand/distributions/range/trait.SampleRange.html). But I guess usage as a bound (`T: SampleRange`) is not affected at least.

Reply:
Oh, so you are saying it is a breaking change anyway? I still think it makes sense to have it deprecated for consistency. As you said, it might possibly keep some code working this way, but I suspect it was not used much.